### PR TITLE
Android 13 fixes

### PR DESCRIPTION
**Pull request by jrtberlin**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -75,4 +75,6 @@
     <string name="cancel">Cancel</string>
     <string name="location_disabled_message">In order to find watches the location service has to be enabled. Once paired you can safely disable it again.</string>
     <string name="location_disabled_title">Location disabled</string>
+    <string name="intro_slide6_title">Phone Notifications</string>
+    <string name="intro_slide6_subtitle">To post notifications on your phone</string>
```
Comment (MagneFire) [inline app/src/main/res/values/strings.xml:79]: Can you maybe add what we use this permission for?
Like: To show the current connection state of the watch.

**Review by MagneFire — COMMENTED** [review]
This PR looks great in general. Thanks for the swift improvements on this!

I know why you added the permission check in https://github.com/AsteroidOS/AsteroidOSSync/pull/178/commits/d1c651c0c27cb83e2a53545a0cb1f069b2540502. It's because of an Android 13 change, but I think it makes sense to describe why we need it in the first place. But see the relevant comment as well.

---

I've tested this on an Android 12 based device and it ran into an exception:
```
2022-10-21 23:41:38.014 2429-2429/org.asteroidos.sync E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.asteroidos.sync, PID: 2429
    java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.fragment.app.Fragment.setMenuVisibility(boolean)' on a null object reference
        at androidx.fragment.app.FragmentStatePagerAdapter.instantiateItem(FragmentStatePagerAdapter.java:194)
        at io.github.dreierf.materialintroscreen.adapter.SlidesAdapter.instantiateItem(SlidesAdapter.java:25)
        at io.github.dreierf.materialintroscreen.widgets.CustomViewPager.addNewItem(CustomViewPager.java:1014)
        at io.github.dreierf.materialintroscreen.widgets.CustomViewPager.populate(CustomViewPager.java:1228)
        at io.github.dreierf.materialintroscreen.widgets.CustomViewPager.populate(CustomViewPager.java:1096)
        at io.github.dreierf.materialintroscreen.widgets.CustomViewPager.onMeasure(CustomViewPager.java:1622)
        at android.view.View.measure(View.java:27129)
        at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:735)
        at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:481)
        at android.view.View.measure(View.java:27129)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7980)
        at androidx.coordinatorlayout.widget.CoordinatorLayout.onMeasureChild(CoordinatorLayout.java:760)
        at androidx.coordinatorlayout.widget.CoordinatorLayout.onMeasure(CoordinatorLayout.java:833)
        at android.view.View.measure(View.java:27129)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7980)
        at android.widget.FrameLayout.onMeasure(FrameLayout.java:197)
        at androidx.appcompat.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:145)
        at android.view.View.measure(View.java:27129)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7980)
        at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1552)
        at android.widget.LinearLayout.measureVertical(LinearLayout.java:842)
        at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
        at android.view.View.measure(View.java:27129)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7980)
        at android.widget.FrameLayout.onMeasure(FrameLayout.java:197)
        at android.view.View.measure(View.java:27129)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7980)
        at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1552)
        at android.widget.LinearLayout.measureVertical(LinearLayout.java:842)
        at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
        at android.view.View.measure(View.java:27129)
        at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7980)
        at android.widget.FrameLayout.onMeasure(FrameLayout.java:197)
        at com.android.internal.policy.DecorView.onMeasure(DecorView.java:1277)
        at android.view.View.measure(View.java:27129)
        at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:4536)
        at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:3228)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3533)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2919)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:10491)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1108)
        at android.view.Choreographer.doCallbacks(Choreographer.java:866)
        at android.view.Choreographer.doFrame(Choreographer.java:797)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1092)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loopOnce(Looper.java:226)
        at android.os.Looper.loop(Looper.java:313)
        at android.app.ActivityThread.main(ActivityThread.java:8663)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
```

Applying the following patches fixes it for me:
```
diff --git a/app/src/main/java/org/asteroidos/sync/PermissionsActivity.java b/app/src/main/java/org/asteroidos/sync/PermissionsActivity.java
index 52dc7ca..b8e763e 100644
--- a/app/src/main/java/org/asteroidos/sync/PermissionsActivity.java
+++ b/app/src/main/java/org/asteroidos/sync/PermissionsActivity.java
@@ -85,7 +85,7 @@ public class PermissionsActivity extends MaterialIntroActivity {
                         Manifest.permission.BLUETOOTH_SCAN) != PackageManager.PERMISSION_GRANTED);
             }
 
-            boolean notificationsPermissionFragmentShown = true;
+            boolean notificationsPermissionFragmentShown = false;
                 SlideFragment notificationsPermissionFragment = null;
             if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
                 SlideFragmentBuilder notificationsPermissionSlideBuilder = new SlideFragmentBuilder()

```

**Comment by jrtberlin** [comment]
Tested on Android 13 and 12

**Comment by MagneFire** [comment]
Looks good now 👍

**Review by MagneFire — APPROVED** [review]

**Comment by OSSCOMAIKABONIFACE** [comment]
Yes Merge It

**Comment by OSSCOMAIKABONIFACE** [comment]
Yes

**Comment by OSSCOMAIKABONIFACE** [comment]
Yes Pls Merge And Remove The MainLog, It Has Been Given Me time To Git Add,
Git Commit And Git Push, There By Hindering My Checkers

**Pull request merged**